src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/ArmorTask.java

```diff
@@ -148,4 +151,35 @@ private boolean isRadioactive(Player p, ItemStack item) {
         return false;
     }
 
+    private boolean shouldProtect(HashedArmorpiece[] armors) {
```

Comment:
"isProtected" sounds better here.

Also since you are reusing this method twice... It may make sense to make a more general method for this inside PlayerProfile.